frontend/src/components/EditGameReview.tsx

```diff
@@ -69,12 +69,15 @@ export default function EditGameReview({gameReview, updateGameReview}: EditGameR
                     onChange={e => {
                         setCategory(e.target.value)
                     }}>
-                    <option>Strategie</option>
-                    <option>RPG</option>
-                    <option>Fantasy</option>
-                    <option>Simulation</option>
-                    <option>Abenteuer</option>
-                    <option>Sport-und-Rennspiel</option>
+                    <optgroup label="=Wähle eine Kategorie=">
```

Comment:
Ist es korrekt, dass hier kein `value` und `onChange` angegeben ist (wie bei `NewGameReview`)?

Reply:
Hey Daniel, ist es doch innerhalb des select-tags zwischen Zeile 68 und 70 oder?

Reply:
Steht in beidem drin.

Reply:
Oh, hubs, stimmt - habe ich ganz übersehen... :)